packages/backend/src/services/auth.ts

```diff
@@ -62,6 +67,14 @@ export class AuthService {
       passwordHash: this._hashService.generatePasswordHash(signUpDto.password),
     };
     const newUser = await this._userRepository.create(createUserDto);
+    await this._verifyService.createVerificationCode(
```

Comment:
Create only phone verification code (using algorithm described above). When phone verified create email verification code with the same algorithm.